### ISSUE TITLE
Support for Contextual Components

### ISSUE THREAD

**Issue by batisteo** [canufeel/ember-leaflet-marker-cluster#5]
It would be nice if this addon would support Contextual Components as well.
Make this happen:

``` handlebars
{{#leaflet-map lat=lat lng=lng zoom=zoom as |layers|}}

  {{tile-layer url="http://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png"}}
  {{#layers.marker-cluster}}
      {{#each markers as |marker|}}
        {{#layers.marker location=marker.location}}
              <h3>{{marker.title}}</h3>
              {{marker.description}}
            {{/layers.marker}}
      {{/each}}
  {{/layers.marker-cluster}}
{{/leaflet-map}}
```

Ref: http://www.ember-leaflet.com/docs/contextual-components

**Comment by canufeel** [comment]
I believe the issue here is that it's not a part of ember-leaflet. https://github.com/miguelcobain/ember-leaflet/blob/master/addon/templates/current/leaflet-map.hbs this template sets up the contextual components support for ember-leaflet. We would have to modify it if we want the support enabled for maker cluster as well. of course it is a possibility. but then we would find ourselves in a situation that if we want some other plugin like `ember-leaflet-some-polyline` in our app and there is another addon for ember-leaflet for that which also overrides this template the plugins would clash. I can't see an easy way to add this kind of support. (

**Comment by cesarizu** [comment]
I think that supporting at least the inner `{{layers.marker}}` tags would be nice, right now you have to use `{{marker-layer}}` instead.

**Comment by miguelcobain** [comment]
Closed by #26
